HOWTO.md

```diff
@@ -324,31 +324,31 @@ cargo install cargo-release
 make dist-rust
 ```
 
-If that doesn't work (consider fixing the make target), otherwise try releasing
-`sbp` and `sbp2json` crates separately, first `sbp`, this will do a dry run
-first:
+If that doesn't work (**status** it don't work, consider fixing the make target),
+otherwise try releasing `sbp` and `sbp2json` crates separately, first `sbp`,
+this will do a dry run first:
 
 ```
-cargo release --package sbp <INCREMENTED_TAG>
+cargo release --exclude sbp2json <INCREMENTED_TAG>
```

Comment:
for some reason including the packages you want doesn't work, but excluding the packages you don' wan does work. 🤷🏻